app/assets/stylesheets/base/top_bar.scss

```diff
@@ -0,0 +1,30 @@
+
+ul {
+  margin-left: 50px;
+  list-style: none;
+  font-size: 20px;
+}
+
+.top-bar {
+  background-color: #15364b;
```

Comment:
Color literals like `#15364b` should only be used in variable declarations; they should be referred to via variable everywhere else.